rest-api/add-gif-picker/README.md

```diff
@@ -0,0 +1,21 @@
+This is an example project to go with our tutorial on How to add a GIF picker to your TalkJS chat.
+<!-- Add link to tutorial once published -->
+
+The project uses TalkJS's [Action Buttons](https://talkjs.com/docs/Features/Customizations/Action_Buttons_Links/) to add a custom action to the message field that lets users add GIFs to their chats. The action is hooked to a backend server that calls the TalkJS REST API to upload the GIF and add it to the conversation.
+
+## Prerequisites
+
+To run this tutorial, you will need:
+
+- A [TalkJS account](https://talkjs.com/dashboard/login)
+- A [Giphy Developer](https://developers.giphy.com/dashboard/) account and an API key
+- [Node.js](https://nodejs.org/en) and [npm](https://www.npmjs.com/)
+
+## How to run the tutorial
+
```

Comment:
Also need a step of getting a Giphy account and pasting the API key in the right place